azure/Guidelines.md

```diff
@@ -993,6 +994,13 @@ Client libraries are required to send telemetry and distributed tracing informat
 - [Azure SDK Distributed tracing policy](https://azure.github.io/azure-sdk/general_azurecore.html#distributed-tracing-policy)
 - [Open Telemetry](https://opentelemetry.io/)
 
+In addition to distributed tracing, Azure also uses a set of common correlation headers:
+
+|Name                         |Applies to|Description|
+|-----------------------------|----------|-----------|
+|x-ms-client-request-id       |Both      |Optional. Caller-specified value identifying the request, in the form of a GUID with no decoration such as curly braces (e.g. `x-ms-client-request-id: 9C4D50EE-2D56-4CD3-8152-34347DC9F2B0`). If the caller provides this header the service **must** include this in their log entries to facilitate correlation of log entries for a single request. Because this header can be client-generated, it should not be assumed to be unique by the service implementation.
```

Comment:
For most services this is a free form string rather than a GUID.  Customers can put whatever they want in here and many use it for crazy things (like tagging so they can grep their logs for analytics rather than use the dozen or ways we make that data so much easier to find).